en/docs/develop/customizations/advanced-ui-customization.md

```diff
@@ -89,6 +91,77 @@ import NewFile from 'AppOverride/src/app/components/Apis/Details/NewFile.jsx';
 
 A compilation error will show up if you try to import the **NewFile.jsx** from **Overview.jsx** as follows.
 
+```sh
```

Comment:
I think the code type should be `javascript` here

Reply:
Fixed

Reply:
Fixed